visualization/dashboard/src/MLIDashboard/FabricStyles.ts

```diff
@@ -81,6 +81,37 @@ export class FabricStyles {
         '#17becf'   // blue-teal
     ];
 
+    // public static fabricColorPalette: string[] = [
```

Comment:
remove commented out code

Reply:
leaving for the purpose of quickly toggling between color palettes until we have a final decision.